hack/verify-goimports.sh

```diff
@@ -18,12 +18,25 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+if [[ -z "${1:-}" ]]; then
+	echo "usage: $0 <path-to-goimports>" >&2
+	exit 1
+fi
+
 goimports=$(realpath "$1")
 
-echo "+++ running goimports"
-output=$("$goimports" -l .)
+# passing "-local" would be ideal, but it'll conflict with auto generated files ATM
+# and cause churn when we want to update those files
+#common_flags="-local github.com/cert-manager/cert-manager"
```

Comment:
note: we should totally be using this flag but it would break on a bunch of files at the moment. I'm leaving that as a task for later.